apps/redi-talent-pool/src/pages/app/browse/BrowseCompany.tsx

```diff
@@ -373,41 +394,37 @@ export function BrowseCompany() {
         )}
       </div>
       <Columns>
-        {jobseekerProfiles
-          ?.filter((profile) => {
-            if (!onlyFavorites) return true
-            const isFavorite =
-              favoritedJobseekersQuery.data?.tpCompanyFavoritedJobseekerProfiles
-                ?.map((p) => p.favoritedTpJobseekerProfileId)
-                ?.includes(profile.id)
-            return isFavorite
-          })
-          .map((profile) => {
-            const isFavorite =
-              favoritedJobseekersQuery.data?.tpCompanyFavoritedJobseekerProfiles
-                ?.map((p) => p.favoritedTpJobseekerProfileId)
-                ?.includes(profile.id)
-
-            if (!isFavorite && onlyFavorites) return
+        {currentItems?.map((profile) => {
+          const isFavorite =
+            favoritedJobseekersQuery.data?.tpCompanyFavoritedJobseekerProfiles
+              ?.map((p) => p.favoritedTpJobseekerProfileId)
+              ?.includes(profile.id)
 
-            return (
-              <Columns.Column
+          return (
+            <Columns.Column
+              key={profile.id}
+              mobile={{ size: 12 }}
+              tablet={{ size: 6 }}
+              desktop={{ size: 4 }}
+            >
+              <JobseekerProfileCard
                 key={profile.id}
-                mobile={{ size: 12 }}
-                tablet={{ size: 6 }}
-                desktop={{ size: 4 }}
-              >
-                <JobseekerProfileCard
-                  key={profile.id}
-                  jobseekerProfile={profile}
-                  linkTo={`/app/jobseeker-profile/${profile.id}`}
-                  toggleFavorite={handleFavoriteJobseeker}
-                  isFavorite={isFavorite}
-                />
-              </Columns.Column>
-            )
-          })}
+                jobseekerProfile={profile}
+                linkTo={`/app/jobseeker-profile/${profile.id}`}
+                toggleFavorite={handleFavoriteJobseeker}
+                isFavorite={isFavorite}
+              />
+            </Columns.Column>
+          )
+        })}
       </Columns>
+      {totalItems > JOBSEEKER_CARDS_PER_PAGE && (
+        <Paginate
```

Comment:
Couldn't name it `Pagination` because we are already importing `<Pagination>` from `pagination.tsx` X) 
This is what `Paginate` returns:
<img width="508" alt="image" src="https://github.com/user-attachments/assets/1ad6ffd8-5b87-4da4-adb0-39e406844f29">

Reply:
That's a common problem with a nice solution: use `as` in the import statements, as such:

![code](https://github.com/user-attachments/assets/b659cc39-4835-439c-8fc1-9f596c6cc732)

Reply:
Sure, if you don't like `Paginate`, we could do that. Or maybe even like this in the `pagination.tsx`:
<img width="450" alt="image" src="https://github.com/user-attachments/assets/07b33344-1174-4556-ac01-eec24519b744">
Personally, I am not a fan of massive imports:) And if we are doing this,  we should stick to this "convention" for all `shadcn` components, I guess, right?

But then, again, we will end up with two `Pagination.tsx` components:
- one in `libs/shared-shadcn-ui-components/src/components/ui/Pagination.tsx` (if we are Pascal-casing it) 
- another one in `libs/shared-atomic-design-components/src/lib/molecules/Pagination.tsx`

Reply:
Yes, we should.

We shouldn't worry about component name collisions in the global namespace; instead, we should focus on each library individually. Name collisions should only be considered within the scope of a single library. This allows us to maintain uniform naming conventions across the codebase without resorting to unnecessary renames like Pagination to Paginate. It keeps things simpler by describing the same notion by one name over two.

Regarding lengthy import statements, they are largely auto-managed and formatted, so they don't require much attention. The Shadcn prefix clearly indicates the source of the components, which I view as an advantage.